examples/other/collection_line.py

```diff
@@ -55,11 +57,17 @@
 
 
 def animate():
+    global t0
     with stats:
         renderer.render(scene, camera, flush=False)
     stats.render()
     canvas.request_draw()
 
+    if t0 is not None:
+        etime = time.perf_counter() - t0
+        print(f"Time to first draw: {etime:0.2f} s")
```

Comment:
we could add this as a feature to the Stats helper pretty easily, to output that number on the console

Reply:
I added it to the renderer - when `show_fps` is set, it also prints the time to the first draw. Also ref #528.